### PR TITLE
add data/ to tar

### PR DESCRIPTION
**Pull request by DouglasBurns**
Allows access to csv files on dice

### CODE REVIEW

**Comment by kreczko** [comment]
This should be already the case with
https://github.com/DouglasBurns/NTupleProduction/blob/e90201ff79a8bdd67f859591988c3b516385f9e3/python/ntp/commands/create/tarball/__init__.py#L73

But of course the path is then `{WORKSPACE}/{CMSSW_SRC}/BristolAnalysis/NTupleTools/data`

**Comment by DouglasBurns** [comment]
So data/ isnt actually added to the tar there then? it should read /BristolAnalysis/NTupleTools/data?

**Comment by kreczko** [comment]
data is linked to `BristolAnalysis/NTupleTools/data`, but in terms of reading in scripts, you would use the WORKSPACE. This way we will not end up with duplicates.

**Comment by DouglasBurns** [comment]
Ok now reading data from BristolAnalysis/NTupleTolls/data. Needs testing first though, which may be a while

File: python/ntp/commands/create/tarball/__init__.py [hunk 1]
```diff
@@ -70,7 +70,9 @@ def __prepare_cmssw_src(self):
         # Note that directories are only looked-for and added under the src/ folder.
         # /data/ subdirs contain data files needed by the code
         # /interface/ subdirs contain C++ header files needed e.g. by ROOT6
-        directories = ['data', 'interface', 'python']
+        directories = [os.path.join(CMSSW_SRC, 'BristolAnalysis', 'NTupleTools','data'), 
+            os.path.join(CMSSW_SRC, 'BristolAnalysis', 'NTupleTools','interface'),
```
Comment (kreczko) [inline python/ntp/commands/create/tarball/__init__.py]: Not what I meant. These paths are already saved in `cmssw_tar`!
What I meant is that in your scripts, please point to the `os.path.join(CMSSW_SRC, 'BristolAnalysis', 'NTupleTools','data'` path. No changes in `create tarball` required!

**Pull request merged**